lib/services/common/domain.js

```diff
@@ -195,11 +195,11 @@ function finishSouthboundTransaction(callback) {
  * @return {Object}             New context containing service information.
  */
 function fillService(context, data) {
-    if (data.service) {
+    if (data && data.service) {
```

Comment:
CNR entry missing? Or is a trivial fix which doesn't need it?

Reply:
CNR already updated

Reply:
Fixed in 5c49c75